src/cmd-buildextend-metal

```diff
@@ -100,6 +100,23 @@ if [ "${ref}" = "null" ]; then
 fi
 commit=$(json_key ostree-commit)
 
+ostree_repo=${tmprepo}
+commitpath=${tmprepo}/objects/${commit::2}/${commit:2}.commit
+commitpartial=${tmprepo}/state/${commit}.commitpartial
+if [ ! -f "${commitpath}" ] || [ -f "${commitpartial}" ]; then
+    # Probably an older commit or tmp/ was wiped. Let's extract it to a separate
+    # temporary repo (not to be confused with ${tmprepo}...) so we can feed it
+    # as a ref (if not temp) to Anaconda.
```

Comment:
```suggestion
    # as a ref (if not temp) to Anaconda.
    echo "Cache for build ${build} is gone"
    echo "Importing commit ${commit} into temporary ostree repo"
```

Reply:
Updated! :arrow_down:  I also tweaked the logic slightly to check the ref itself, since the commit might be in the repo, but if the ref is pointing to something else, then we'd be installing the wrong thing.